src/main/flight/navigation_rewrite.c

```diff
@@ -1626,7 +1633,7 @@ bool isLandingDetected(void)
         landingDetected = isFixedWingLandingDetected(&landingTimer);
     }
     else {
-        landingDetected = isMulticopterLandingDetected(&landingTimer, &hasHadSomeVelocity);
+        landingDetected = isMulticopterLandingDetected(&landingTimer, &hasHadSomeVelocity, &landingThrSum, &landingThrSamples);
```

Comment:
I suggest moving all these parameters to a `typedef struct` to make things cleaner.